include/deal.II/fe/fe.h

```diff
@@ -149,6 +149,65 @@ template <int dim, int spacedim> class FESystem;
  * FiniteElement::system_to_block_index(). The number of blocks of a finite
  * element can be determined by FiniteElement::n_blocks().
  *
+ * To better illustrate these concepts, let's consider the following
+ * example
+ * @code
+ * FESystem<dim> fe_basis(FE_Q<dim>(2), dim, FE_Q<dim>(1),1);
+ * @endcode
+ * with <code>dim=2</code>. The resulting finite element has 3 components:
+ * two coming from the quadratic element and one from the linear element. In case
+ * of fluids, one can think of the first two components representing a
```

Comment:
"If, for example, this system were used to discretize a problem in fluid dynamics then one could think..."